cli/bin/grain.js

```diff
@@ -1,4 +1,4 @@
-#!node --harmony
+#!/usr/bin/env node
```

Comment:
So two issues were occurring here on Ubuntu. 

First off, `#!node` didn't see to work on its own, I added the `#!/usr/bin/env node` as I believe this is the most common way to handle shebangs for node.

Secondly, the `--harmony` was not playing nicely with the above command, and I did some digging, and there does not seem to be a standard for if/how shebangs support flags. However, I didn't really see the use case for it at all. It seems to run fine on node versions `8`, `9`, and `10`.

Please let me know if this is a problem.

Reply:
cc: @ospencer 
I have to test whether this works for me later this evening, but I see no issues (in fact, I would have probably complained had I noticed `node --harmony` originally....but I digress.)

Reply:
Oh, yep. That's definitely one of those testing things that got merged. 🙂 

This should definitely be fine, but we should make sure that we have a note in the README about having `node` >= 8.

Reply:
@ospencer I can add that note on #38 if you'd like.

Reply:
Though actually now that I think about it, we don't need the shebang at all, since this script is always invoked through the node tooling. I remember adding that in specifically because I was trying to do something with some harmony feature (which never ended up happening).

Reply:
@ospencer I think we still do need it. I just removed it, and it was not happy:

```sh
root@ad15e8007768:~# grain
/usr/local/bin/grain: line 1: syntax error near unexpected token `('
/usr/local/bin/grain: line 1: `let program = require('commander');'
```

Reply:
Alright! I'm down with it.